openapi/endpoints/kyc/get-kyc-profile.yaml

```
@@ -0,0 +1,26 @@
+tags:
+  - kyc
+summary: Get a KYC profile
```

```suggestion
summary: Retrieve KYC profile
```

This is consistent with our other KYC endpoints - Non blocking though